ntoskrnl/config/cmsysini.c

```diff
@@ -1144,7 +1144,6 @@ CmpCreateRegistryRoot(VOID)
                             &CmpRegistryRootHandle);
     if (!NT_SUCCESS(Status))
     {
-        ObDereferenceObject(RootKey);
```

Comment:
Right. Moreover, it needs to be added at line 1112.

Reply:
Could you please elaborate on why? As I see ObReferenceObjectByHandle is called on line 1151, and no reference occurs before 1112

Reply:
@wnstngs Isn't the ObCreateObject giving the initial reference? (the object now exists after all...)

Reply:
For the same reason as old lines 1124 and 1160.
That `ObReferenceObjectByHandle()` doesn't matter for this PR.